huaweicloud/services/acceptance/projectman/resource_huaweicloud_projectman_project_test.go

```diff
@@ -0,0 +1,102 @@
+package projectman
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getProjectResourceFunc(config *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	var (
+		getProjectHttpMethod = strings.ToUpper("GET")
```

Comment:
This seems a bit werid, can we get rid of calling ToUpper to a fixed string.

Reply:
fix